### PR TITLE
Sprint 3/issue 7#company jobdetailsscreen view

### PR DESCRIPTION
**Pull request by fpreciado3**
# Description

- Add the Job Details Screen so the user can look at job offers in a detailed level
- Change the data files to include the data needed in this screen and component

## Testing

Null

## Screenshots

image

### CODE REVIEW

**Review by VISOSO2403 — APPROVED** [review]
Good job 🥵

**Review by Antonio3veces3 — APPROVED** [review]
Well done!

**Review by Alfredo20164920 — APPROVED** [review]

File: src/data/CompanyWorkData.ts [hunk 1]
```diff
@@ -1,37 +1,130 @@
...
+    //     ],
+    //     benefits: [
+    //         { item: 'Competitive salary and benefits package'},
+    //         { item: 'Opportunities for career growth and advancement within PepsiCo'},
+    //         { item: 'Training and development programs to enhance your skills and knowledge'},
+    //         { item: 'Employee discounts on Pepsi products and other company offerings'},
+    //         { item: 'Health and wellness programs to support your overall well-being'},
+    //         { item: 'A supportive and inclusive work environment that values diversity and teamwork'},
+    //     ],
+    // },
```
Comment (juperameza) [inline src/data/CompanyWorkData.ts:127]: Don't leave commented code

File: src/interface/companyworkinterface.ts [hunk 2]
```diff
@@ -5,7 +5,14 @@ export interface CompanyWork {
...
     salary:string;
+    salaryType:string;
     date:string;
     time:string;
+    companyOverview:string;
+    jobDescription:string;
+    keyResponsibilities:string[any];
+    qualifications:string[any];
+    benefits:string[any];
 }
```
Comment (juperameza) [inline src/interface/companyworkinterface.ts:18]: New line at the end of the file

File: src/screens/WorkDetailScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        );
+};
+
+const styles = StyleSheet.create({
+    container: {
+        margin:16
+    },
+});
+
+export default WorkDetailScreen;
```
Comment (juperameza) [inline src/screens/WorkDetailScreen.tsx:30]: New line at the end of the file

**Review by juperameza — CHANGES_REQUESTED** [review]

**Pull request merged**